### DISCUSSION TITLE
Till what max speed can we push UVTools acceleration gcode without damaging the printer ?

### DISCUSSION THREAD

**Discussion by Alomphega** [sn4k3/UVtools#487] [Q&A] [answered]
Hello,

Since I would like to improve my printing time, I wonder until how many I can set the max speed within the uvtools_acceleration.gcode file instead of the original 240 mm/mn ?

Do you think I can try with 480 mm/mn for a Saturn ? More than this ? Less ?

Cordially,
Guy

**Comment by sn4k3** [comment]
You can try more speed, but i think its worthless.
Because acceleration is low, and may never reach 240mm/min anyway because small lifts, in other words the require time to reach max speed is superior to your actual lift and then you may never see 240mm/min unless you use large lift height.
Observe the normal printing lift. Then from top send machine to home, does home Z movement reach top speed while print lift dont?

What you can do is increase acceleration, too bad they dont support decimals, so I1 would go to I2, and thats the double of acceleration.
UVtools values are the most conservative values, you can and may tune.

**Reply by Alomphega** [reply]
I guessed that. So how to do increase acceleration within UVTools ?

**Reply by sn4k3** [reply]
Not with UVtools but with .gcode file.
Open the file, read the comments and adjust.
Start by acceleration and increase from there.
Do some dry prints to test

**Reply by Alomphega** [reply]
> UVtools values are the most conservative values, you can and may tune.

That's why I asked how to do in UVTools... Well, don't worry, I'll try incrementing acceleration value within your gcode acceleration file tomorrow I guess. See you later !

**Comment by Alomphega** [comment]
Hello,
Please I need information : 
Within the Saturn original parameters gcode we have this value and these comments : 
« M8008 I900      ; Acceleration, the larger the value, the larger the average speed of actual operation, but the noise is also large, the value is small, and the actual speed will be smaller. »
But in your acceleration gcode file the value is : 
« M8008 I10  ; Acceleration mm/s2 »
These "900" and "10" values are 900 mm per second and 10 mm per second ? 
Why this huge difference ?
What will arrive if I set "900" instead "10" for the M8008 parameter in your acceleration gcode file ?

**Reply by sn4k3** [reply]
> These "900" and "10" values are 900 mm per second and 10 mm per second ?

Correct, but in mm per second square

> Why this huge difference ?

Because they dont know what they are doing, mostly these are parameters from FDM printers and represent a huge acceleration that almost hit the target speed in same instant

> What will arrive if I set "900" instead "10" for the M8008 parameter in your acceleration gcode file ?

It will accelerate 900mm per second squared and then you will lose the smooth effect.

IMO go in 10 increments and run dry tests, easy and fast to test

**Reply by Alomphega** [reply]
> It will accelerate 900mm per second squared and then you will lose the smooth effect.

Exactly what I just experimented !

> IMO go in 10 increments and run dry tests, easy and fast to test

Absolutely what I will do. 
Thank you very much for all these explanations, dear Master come from a so far galaxy ! ;-)

**Reply by Alomphega** [reply]
> IMO go in 10 increments and run dry tests, easy and fast to test

I've tried some settings, and you know what ? Just push "M8008 I" to 20 mm/s² instead of 10, 
and my printing time did come from 06h54mn to 05h48mn ! Perfect for me.

Thank you very much Tiago 👍😊

**Reply by sn4k3** [reply]
And how did it affect the movement? Faster and less smooth but still in a good margin?

**Reply by Alomphega** [reply]
Exactly : faster and a bit less smooth (almost not visible difference) but still in a good margin. 

I guess I could use 30 (and maybe even 40) before canceling the soft acceleration effect. But in this case, maybe the up could be too much quick, and the parts would come off the board to stay on the FEP because of the suction.

**Reply by sn4k3** [reply]
All a matter of try out ;)

**Comment by Alomphega** [comment]
I was pretty happy, ready to come back here to say "Hurra, 120 mm/s² acceleration in gcode file does the job !" BUT alas, if I tried with success on 2 Saturn printers, the third did not print all the parts (3 parts/8 did stay on the FEP).
Because this is the printer which require +0,4mm on Z=0 I guess this gap is guilty : more resin to cure, so a bit less grip on build plate, a bit too much acceleration, a bit fail. 
Well, I will re-do further tries to be sure before to tell here what will be the right max acceleration I can trust on Saturn printers.
See you later !

**Reply by sn4k3** [reply]
120 mm/s² / 12cm/s² looks too much already.
With UVtools you can ajust the starting Z height to counter that offset (If offset is positive)
![UVtools_2022-06-02_15-17-46](https://user-images.githubusercontent.com/113281/171650434-a261333f-290b-4003-842d-22db27d86f62.png)

**Reply by Alomphega** [reply]
120mm/s² was good enough on 2 printers with +0,2mm on Z=0... I guess the smooth is more important than the speed itself, and even if the smooth is quick, it can do the job. BUT of course, if the printer have a too big miss, its have more chances to fail.
Well, I'm trying 60mm/s² and it seems good also for my +0,4mm Z=0 one. I cross my finger while the last hour is printing now.

The +0,4mm I have to add on the Z=0 is an Elegoo advise, and it is set directly within the printer which need it. I prefer using UVTools to make one same file for all my printers, and set the offset differences required by printers directly on each ones.

**Comment by Alomphega** [comment, marked as answer]
I have finally the answer : 60 mm/s² is that good ! 😊👍

**Reply by Alomphega** [reply]
Hello,
Just an update to say I have downed the acceleration speed to 40 mm/s² to keep a good margin.
And to say thanks to UVTools, I succeeded my first "not any issue" production series !
Thank you very much Tiago.
Best Regards,
Guy

**Reply by sn4k3** [reply]
Seens a reasonable value 👍
Keep up the good work!
